loganalyzer/custom_analyzers/ShootAnalyzer.py

```diff
@@ -0,0 +1,169 @@
+from datetime import datetime
+
+class ShootAnalyzer:
+    def __init__(self, game, log_name):
+        self.game = game
+        self.log_name = log_name
+        self.left_goal_count = 0
+        self.right_goal_count = 0
+        self.cycles_before_goal = 50
+        self.shoots_list = []
+        self.timestamp = datetime.now().strftime("%Y-%m-%d %H:%M:%S")
```

Comment:
Qual que é a motivação de colocar o nome do log e o timestamp no CSV? É pra depois organizar melhor na interface? Se for isso, também é interessante que façamos isso para os outros analyzers?

Reply:
O analyzer dos chutes pega pra analisar todos os logs de uma pasta, e aí o nome do log é pra identificar de qual jogo que é aquele chute (na interface daria para exibir os chutes de um jogo ou de todos os jogos). O timestamp é só para se você sem querer rodar o analyzer duas vezes e tiver dados repetidos conseguir ver que isso aconteceu e apagar esses dados (no final talvez isso não aconteça, mas deixei só por garantia)